test/smoke/testApps/OpenTelemetryApiSupport/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'war'
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+compileSmokeTestJava.sourceCompatibility = 1.8
+compileSmokeTestJava.targetCompatibility = 1.8
+
+war {
+    // this is done to remove the version from the archive file name
+    // to make span name verification simpler
+    archiveFileName = project.name + '.war'
+}
+
+ext.testAppArtifactDir = war.destinationDirectory
+ext.testAppArtifactFilename = project.name + '.war'
+
+dependencies {
+    compileOnly aiAgentJar // this is just to force building the agent first
+
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: '2.1.7.RELEASE') {
+        exclude group: 'org.springframework.boot', module: 'spring-boot-starter-tomcat'
+    }
+
+    compile group: 'io.opentelemetry', name: 'opentelemetry-api', version: '0.10.0'
```

Comment:
do you want to test 0.11.0 based on the github wiki page you created?

Reply:
good point, this branch was still on 0.10.0, I merged master into this branch and updated the test to 0.11.0